lib/tasks/redirect_export_health_certificates.rake

```diff
@@ -0,0 +1,141 @@
+    "/export-health-certificates/export-canned-pet-food-to-or-transit-through-the-european-union-certificate-8324",
+    "/export-health-certificates/export-dog-chews-to-the-european-union-certificate-8325",
+    "/export-health-certificates/export-flavouring-innards-for-use-in-the-manufacture-of-pet-food-to-the-european-union-certificate-8326",
+    "/export-health-certificates/export-fresh-or-chilled-hides-and-skins-of-ungulates-to-the-european-union-certificate-8327",
+    "/export-health-certificates/export-treated-hides-and-skins-of-ungulates-to-or-transit-through-the-european-union-certificate-8328",
+    "/export-health-certificates/export-treated-hides-and-skins-of-ruminants-and-equidae-that-have-been-separated-for-21-days-to-the-european-union-certificate-8329",
+    "/export-health-certificates/export-of-game-trophies-or-other-preparations-of-birds-and-ungulates-to-the-european-union-certificate-8330",
+    "/export-health-certificates/export-pig-bristles-from-third-countries-free-from-african-swine-fever-to-the-european-union-certificate-8331",
+    "/export-health-certificates/export-fish-oil-not-intended-for-human-consumption-to-or-transit-through-the-european-union-certificate-8332",
+    "/export-health-certificates/export-processed-animal-protein-pap-other-than-types-derived-from-farmed-insects-to-the-european-union-certificate-8333",
+    "/export-health-certificates/movement-back-to-the-european-union-of-registered-horses-for-racing-competition-and-cultural-events-after-temporary-import-to-the-uk-for-a-period-of-less-than-30-days-certificate-8334",
+    "/export-health-certificates/permanently-export-an-equine-animal-for-breeding-or-production-to-the-european-union-certificate-8335",
+    "/export-health-certificates/temporarily-export-a-registered-horse-for-a-period-of-less-than-90-days-to-the-european-union-certificate-8336",
+    "/export-health-certificates/export-a-live-equine-animal-destined-for-another-third-country-which-will-transit-the-european-union-certificate-8337",
+    "/export-health-certificates/export-wild-leporidae-farmed-rabbits-and-wild-land-mammals-other-than-ungulates-for-the-transit-through-or-storage-in-the-european-union-certificate-8339",
+    "/export-health-certificates/export-intermediate-products-to-be-used-for-the-manufacture-of-medicinal-and-cosmetic-products-intended-for-dispatch-to-or-for-transit-through-the-european-union-certificate-8341",
+    "/export-health-certificates/export-bone-horn-and-hoof-products-excluding-meal-intended-for-use-other-than-as-feed-material-organic-fertilisers-or-soil-improvers-for-dispatch-to-or-for-transit-through-the-european-union-certificate-8342",
+    "/export-health-certificates/export-untreated-wool-and-hair-to-the-european-union-certificate-8343",
+  ]
+  redirect_path = "/starting-to-export/licences"
+
+  paths.each do |path|
+    content_id = Services.publishing_api.lookup_content_ids(base_paths: path, exclude_unpublishing_types: [])
```

Comment:
You might be able to call this just once with all paths. That should return a hash from path to content_id for all the paths.